l10n_nl_xaf_auditfile_export/views/templates.xml

```diff
@@ -148,7 +148,7 @@
                         <docRef><t t-esc="l.ref" t-esc-options='{"widget": "auditfile.string999"}' /></docRef>
                         <effDate><t t-esc="l.date" /></effDate>
                         <desc><t t-esc="l.name" /></desc>
-                        <amnt><t t-esc="l.credit or l.debit" /></amnt>
+                        <amnt><t t-esc="round(l.credit or l.debit, 2)" /></amnt>
```

Comment:
I think you don't need to round here. The risk is that the rounding will generate an amount different than `l.balance`. See https://github.com/OCA/l10n-netherlands/pull/130#issuecomment-353287014

Reply:
Well we had this error:
```
:507419:0:ERROR:SCHEMASV:SCHEMAV_CVC_FRACTIONDIGITS_VALID: Element '{http://www.auditfiles.nl/XAF/3.2}amnt': [facet 'fractionDigits'] The value '0.00999999999999' has more fractional digits than are allowed ('2').
```

Reply:
Isn't abs(l.balance) fixing that issue as well?

Reply:
Yes, it works, thanks.